nc-news/src/components/Article.jsx

```diff
@@ -1,8 +1,55 @@
+import axios from 'axios'
+import { useEffect, useState } from 'react'
+
 function Article(){
+    const [loading, setLoading] = useState(true)
+    const [article, setArticle] = useState([])
+    
+    const article_id = 5
```

Comment:
Do you need to hard code an article_id for a specific reason? This can be deleted 😄